#Request
#Empty
---
#Response
int32 result
